js/background.js

```diff
@@ -14,6 +14,7 @@ define(function(require) {
 	'use strict';
 
 	var OC = require('OC');
+	var BackGround = this;
```

Comment:
> Seems like travis failed because you are not using _this(?)

Yes indeed. I'll have to fix that. I named it 'Background' since it's more or less a static class.